packages/material-ui/src/Button/Button.test.js

```diff
@@ -370,150 +370,4 @@ describe('<Button />', () => {
       expect(markup.text()).to.equal('Hello World');
     });
   });
-
-  describe('custom theme variants', () => {
```

Comment:
This is already covered in `useThemeVariants.test.js`,  no need for per component tests